angular-ngc/libraries/lib-a/src/lib/strings/BUILD.bazel

```diff
@@ -3,5 +3,6 @@ load("//tools:ts.bzl", "ts_project")
 ts_project(
     name = "strings",
     srcs = glob(["*.ts"]),
+    supports_workers = False,
```

Comment:
?

Reply:
I am using WSL 2 under Windows, is that the reason?

Reply:
You can revert this after rebasing

Reply:
done.